test/com/linkedin/parseq/TestTask.java

```diff
@@ -1,16 +1,34 @@
 package com.linkedin.parseq;
 
 import com.linkedin.parseq.promise.Promises;
+import java.util.Arrays;
 import java.util.concurrent.TimeoutException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static org.testng.Assert.*;
+
 
 /**
  * @author Zhenkai Zhu
  * @author Jaroslaw Odzga (jodzga@linkedin.com)
  */
 public class TestTask extends AbstractTaskTest {
 
+  private boolean _crossThreadStackTracesEnabled;
```

Comment:
That I don't know. I only used it with BeforeMethod. Unfortunately I did not find an official documentation about this.